archive-bibliotech.php

```diff
@@ -14,7 +14,7 @@
  *
  * @package WordPress
  * @subpackage Twenty_Twelve
- * @since Twenty Twelve 1.0
+ * @since 1.0
```

Comment:
Okay, sounds good...

I'll create one.

2016-07-20 12:34 GMT-04:00 Matthew Bernhardt <notifications@github.com>:

> In archive-bibliotech.php
> <https://github.com/MITLibraries/MITLibraries-news/pull/69#discussion_r71559917>
> :
>
> > @@ -14,7 +14,7 @@
> >   *
> >   * @package WordPress
> >   * @subpackage Twenty_Twelve
> > - * @since Twenty Twelve 1.0
> > + * @since 1.0
>
> Is there a separate PR to remove unneeded templates? If so, then great. If
> not, let's create one. At the end of this, anything that exists should have
> a proper FileComment block - so it either needs to be deleted, or properly
> commented.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MITLibraries/MITLibraries-news/pull/69/files/681ea9151eb773f0fde949e71eccd585b468c1e9#r71559917>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABv1VcnYRZs8Y-PwVnI4QwUkuUSuxUsqks5qXk37gaJpZM4JLyGJ>
> .
>



-- 

"The reason is never the reason."


*- Some French Movie Director*"Every portrait that is painted with feeling
is a portrait of the artist, not the sitter."
*- Oscar Wilde*